lib/tizen_plugins.dart

```diff
@@ -405,9 +409,18 @@ Future<void> _informAvailableTizenPlugins(FlutterProject project) async {
   final List<String> plugins = (await findPlugins(project)).map((Plugin p) => p.name).toList();
   for (final String plugin in plugins) {
     final String tizenPlugin = '${plugin}_tizen';
-    if (_kKnownPlugins.contains(plugin) && !plugins.contains(tizenPlugin)) {
-      globals.printWarning(
-          '$tizenPlugin is available on pub.dev. Did you forget to add to pubspec.yaml?');
+    if (_kKnownPlugins.containsKey(plugin) &&
+        !plugins.contains(tizenPlugin) &&
+        !_kKnownPlugins[plugin]!
+            .any((String recommendedPlugin) => plugins.contains(recommendedPlugin))) {
+      final List<String> recommendedPlugins = _kKnownPlugins[plugin]!;
+      if (recommendedPlugins.isEmpty) {
+        globals.printWarning(
+            '$tizenPlugin is available on pub.dev. Did you forget to add to pubspec.yaml?');
+      } else {
+        globals.printWarning(
+            '[${recommendedPlugins.join(', ')}] are available in pub.dev. Did you forget to add to pubspec.yaml?');
```

Comment:
```suggestion
            '[${recommendedPlugins.join(', ')}] are available on pub.dev. Did you forget to add to pubspec.yaml?');
```
The message ("Did you forget to add") is not very clear. Does it mean the user should add all the listed packages to `pubspec.yaml`?

Reply:
I udpated code for more clear message.
````
[webview_flutter_tizen or webview_flutter_lwe] are available on pub.dev. Did you forget to add to pubspec.yaml?
```

Reply:
nit: are should be is

Reply:
Thanks, done.